packages/react/src/dropdown-menu/DropdownMenuDivider.tsx

```diff
@@ -0,0 +1,20 @@
+import { boxPalette, tokens } from '../core';
+
```

Comment:
Perhaps this could be based on the Divider component?

Reply:
Done ✅